index.html

```diff
@@ -0,0 +1,26 @@
+<html>
+<head>
+    <title>HPO</title>
+</head>
+<body>
+    <h1 style="text-align: center;font-family: Arial, Helvetica, sans-serif;"><u>HPO as a Service</u> (HPOaaS)</h1>
+    
+    <p style="font-size:18px; font-family: Arial, Helvetica, sans-serif;"> HPO as a service helps the user find the optimal configuration values for any model. </p>
+    <h2 style="font-family: Arial, Helvetica, sans-serif;">Goal:</h2>
+    <p style="font-size:18px; font-family: Arial, Helvetica, sans-serif;">
+        Provide Hyperparameter Optimization(HPO) as a service to choose the optimal hyperparameters provided by the user for any model. 
+        Primary use case is to optimize the performance of an environment by tuning below layers independently or together.
+            * Container : resource usage like cpu and memory requests /limits.
+            * Runtime: Ex : hotspot, openj9, nodeJS , etc.
+            * Application stack:  Ex: Quarkus , Liberty, postgres etc.
+            * OS layer.
```

Comment:
Please remove this as this is not needed

Reply:
Done